daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -355,8 +355,8 @@ class TransactionCoderSpec
 
     "fail if try to encode a create node containing value with version different from node" in {
```

Comment:
without the change in `ValueGenerator#transactionVersionGen` the two tests of this file have not the expected coverage.